man-roxygen/path.R

```diff
@@ -1,4 +1,9 @@
 #' @param path Drive File, something that identifies where the file should live
 #'   on your Google Drive. Can be an actual path (character), a file id marked
-#'   with \code{\link{as_id}()}, or a \code{\link{dribble}}.
+#'   with \code{\link{as_id}()}, or a \code{\link{dribble}}. If specified as an
+#'   actual path, you can include the name, indicated by not appending a final
+#'   slash (i.e.: "abc/def" would indicate the file will be named "def" and placed
+#'   in "abc", where as "abc/def/" indicates the file will be placed in folder "def"
+#'   which is located in folder "abc"), or you can specify the name with the
+#'   \code{name} parameter.
```

Comment:
My explanation for `path` has gotten quite wordy 😬 -- happy if you have any suggestions @jennybc  to have it be understandable but brief!